docs/docs/getting-started/document-qa.mdx

```diff
@@ -0,0 +1,88 @@
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import ZoomableImage from "/src/theme/ZoomableImage.js";
+import ReactPlayer from "react-player";
+import Admonition from "@theme/Admonition";
+
+# Document QA
+
+Build a question-and-answer chatbot with a document loaded from local memory.
+
+## Prerequisites
+
+1. Install Langflow.
+```bash
+pip install langflow
```

Comment:
Hey @mendonk 

This should be `python -m pip install langflow --pre` because these aren't available in the main version.